packages/internal/src/build/babel/common.ts

```diff
@@ -62,3 +68,65 @@ export const getCommonPlugins = () => {
     ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
   ]
 }
+
+/**
+ * Finds, reads and parses the [ts|js]config.json file
+ * @returns The config object
+ */
+export const parseConfigFiles = () => {
```

Comment:
In line with the other comments, let's be more explicit about which config files:

```suggestion
export const parseTypeScriptConfigFiles = () => {
```

Reply:
this method can also parse `jsconfig.json` files, should we rename it?

Reply:
See https://github.com/redwoodjs/redwood/pull/7575#discussion_r1196940890